servers/audio_server.h

```diff
@@ -230,8 +274,15 @@ class AudioServer : public Object {
 	void init_channels_and_buffers();
 
 	void _mix_step();
+	void _mix_step_for_channel(AudioFrame *p_out_buf, AudioFrame *p_source_buf, AudioFrame p_vol_start, AudioFrame p_vol_final, float p_attenuation_filter_cutoff_hz, float p_highshelf_gain, AudioFilterSW::Processor *p_processor_l, AudioFilterSW::Processor *p_processor_r);
+
+	// Should only be called on the main thread.
+	AudioStreamPlaybackListNode *_find_playback_list_node(Ref<AudioStreamPlayback> p_playback);
 
 	struct CallbackItem {
+		std::atomic<CallbackItem *> next;
```

Comment:
In this case, since we don't have a `SafePointer<T>`, using `std::atomic` directly is fine. If we find many cases of atomic pointers sometime in the future, we can always introduce it.

In any case, again, it'd be good to check if acquire-release (or even something weaker) can be used intead of sequential in the operations.

Reply:
Mentioned above, but I'm not entirely comfortable switching over to acquire release here. I'm not convinced it would be a problem, but I can't convince myself it's safe either.

Reply:
Wow nevermind! These atomics can be removed. They're dead code leftover from before I factored out SafeList into its own file.